packages/babylon/test/fixtures/typescript/type-arguments/tagged-template-no-asi/output.json

```diff
@@ -0,0 +1,184 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 11,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 2,
+      "column": 2
+    }
+  },
+  "program": {
+    "type": "Program",
+    "start": 0,
+    "end": 11,
+    "loc": {
+      "start": {
+        "line": 1,
+        "column": 0
+      },
+      "end": {
+        "line": 2,
+        "column": 2
+      }
+    },
+    "sourceType": "module",
+    "body": [
+      {
+        "type": "ExpressionStatement",
+        "start": 0,
+        "end": 11,
+        "loc": {
+          "start": {
+            "line": 1,
+            "column": 0
+          },
+          "end": {
+            "line": 2,
+            "column": 2
+          }
+        },
+        "expression": {
+          "type": "TaggedTemplateExpression",
+          "start": 0,
+          "end": 11,
+          "loc": {
+            "start": {
+              "line": 1,
+              "column": 0
+            },
+            "end": {
+              "line": 2,
+              "column": 2
+            }
+          },
+          "tag": {
+            "type": "NewExpression",
```

Comment:
This looks wrong. The tagged template should have higher precedence than the *NewExpression*. As an example, in the following:

```js
new C
``
```

here's what I currently get:

```json5
{
  // ...
  "expression": {
    "type": "NewExpression",
    "start": 0,
    "end": 8,
    "loc": {
      "start": {
        "line": 1,
        "column": 0
      },
      "end": {
        "line": 2,
        "column": 2
      }
    },
    "callee": {
      "type": "TaggedTemplateExpression",
      "start": 4,
      "end": 8,
      "loc": {
        "start": {
          "line": 1,
          "column": 4
        },
        "end": {
          "line": 2,
          "column": 2
        }
      },
      "tag": {
        "type": "Identifier",
        "start": 4,
        "end": 5,
        "loc": {
          "start": {
            "line": 1,
            "column": 4
          },
          "end": {
            "line": 1,
            "column": 5
          },
          "identifierName": "C"
        },
        "name": "C"
      },
      "quasi": {
        "type": "TemplateLiteral",
        "start": 6,
        "end": 8,
        "loc": {
          "start": {
            "line": 2,
            "column": 0
          },
          "end": {
            "line": 2,
            "column": 2
          }
        },
        "expressions": [],
        "quasis": [
          {
            "type": "TemplateElement",
            "start": 7,
            "end": 7,
            "loc": {
              "start": {
                "line": 2,
                "column": 1
              },
              "end": {
                "line": 2,
                "column": 1
              }
            },
            "value": {
              "raw": "",
              "cooked": ""
            },
            "tail": true
          }
        ]
      }
    },
    "arguments": []
  }
}
```